src/source/feeds.rst

```diff
@@ -1,916 +1,217 @@
-String identifier max 70 chars.
+        :download:`download example<examples/feed-xml-v-1-0-example.xml>`
 
-======= ============================================
-Example .. code-block:: html
-        
-         <ad:brand>iPhone</ad:brand>
-======= ============================================
+    .. collapse:: Example v.1.1
 
-.. index:: gtin
-.. _feed_gtin: 
+        .. literalinclude:: examples/feed-xml-v-1-1-example.xml
+            :language: xml
 
-GTIN
-""""""""""""""""""""""
+        :download:`download example<examples/feed-xml-v-1-1-example.xml>`
 
-GTIN (Your product’s Global Trade Item Number), definition follows `Google Merchant Center <https://support.google.com/merchants/answer/6324461>`__ guidelines.
-String identifier max 50 chars.
+    |
 
-======= ==================================
-Example .. code-block:: html
-        
-         <ad:gtin>44320194113475</ad:gtin>
-======= ==================================
+.. collapse:: TSV
 
-.. index:: itemGroupId
-.. _feed_itemGroupId: 
+    TSV format follows specification described on `wiki <https://en.wikipedia.org/wiki/Tab-separated_values>`__.
+    Column names used in TSV feed must superset :ref:`column names <feed-details>` specified.
+    Some complex fields, like :ref:`feed_ship` or :ref:`feed_attr` must follow the specified encoding conventions.
 
-itemGroupId
-""""""""""""""""""""""
+    :download:`download example<examples/feed-tsv-example.tsv>`
 
-Item Group Id definition follows `Google Merchant Center <https://support.google.com/merchants/answer/6324507>`__ guidelines.
-String identifier max 1-50 chars.
+|
 
-======= ============================================
-Example .. code-block:: html
-        
-         <ad:itemGroupId>BC23456</ad:itemGroupId>
-======= ============================================
-
-.. index:: condition
-.. _feed_condition: 
-
-condition
-""""""""""""""""""""""
-
-Condition definition follows `Google Merchant Center <https://support.google.com/merchants/answer/6324469>`__ guidelines.
-Accepted values: *new*, *refurbished*, *used*
-
-======= ==================================
-Example .. code-block:: html
-        
-         <ad:condition>used</ad:condition>
-======= ==================================
+Errors
+------
+:ref:`get_feed_import` shows the overview of imported feeds of the user, and
+:ref:`get_feed_import_id_detail` can be a good source of information on possible errors
+or warnings on ads, such as missing mandatory attributes or too low CPC etc. 
 
-.. index:: material
-.. _feed_material: 
 
-material
-""""""""""""""""""""""
+.. _feeds_qna:
 
-Material definition follows `Google Merchant Center <https://support.google.com/merchants/answer/6324410>`__ guidelines.
-String identifier max 200 chars.
-  
-======= ==================================
-Example .. code-block:: html
-        
-         <ad:material>Wool</ad:material>
-======= ==================================
+Scenario's & Questions
```

Comment:
perhaps an additional section, that says 'I have a google feed, how do I modify it properly for Admarkt'?

Reply:
Kieran has prepared onboarding instructions. I think it is exactly the answer to that question. We will add it.